contracts/interfaces/0.8.x/IMintingAPIMinterV0.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/* Provides an interface definition for compatibility with the Minting API.
+ *
+ * In order for a minter contract to be compatible with the REST API provided
+ * at https://minting-api.artblocks.io/ it must implement the interface defined
+ * below.
+ */
+interface IMintingAPIMinterV0 {
+    mapping(uint256 => bool) public projectMaxHasBeenInvoked;
```

Comment:
mappings aren't allowed in an interface, but a public mapping actually creates the following public getter function. We label it as external here since interfaces must have all functions defined external (but contracts can define a public function and still meet the requirement of the interface)
```suggestion
    function projectMaxHasBeenInvoked(uint256) external returns (bool);
```